@kiva/kv-components/src/vue/KvWwwHeader/KvHeaderLinkBar.vue

```diff
@@ -39,6 +39,7 @@
 			:open-menu-item="openMenuItem"
 			:on-hover="onHover"
 			:dropdown-icon="mdiChevronDown"
+			send-link-position
```

Comment:
Not specific to this changeset, but I think we can reduce the size of the titles and remove some of the white space between accordions in mobile:

<img width="462" height="267" alt="Image" src="https://github.com/user-attachments/assets/54fbec10-7097-4c84-844f-379e18c3adb5" />